### ISSUE TITLE
attribute 'recurseIntoAttrs' missing

### ISSUE THREAD

**Issue by ezemtsov** [musnix/musnix#123]
Musnix complains about missing attribute

Nix config:
```
  musnix = {
    enable = true;
    kernel = {
      optimize = true;
      realtime = true;
    };
  };
```

Error:
```
~> sudo nixos-rebuild switch
building Nix...
building the system configuration...
error: attribute 'recurseIntoAttrs' missing, at /etc/nixos/musnix/overlay.nix:172:27
(use '--show-trace' to show detailed location information)
```

Nix version:
```
~> nixos-version
20.03.2668.7c85b915892 (Markhor)
```

**Comment by henrytill** [comment]
Interesting.  I haven't tested against 20.03 recently.  I'll set up a VM and have a look.

**Comment by ezemtsov** [comment]
Running same on `nixos-unstable` (20.09) seem to work. Nix builds the rt linux kernel, but it fails on boot with `bug using smp_processor_id() in preemptible code` (probably another issue?)

I tried to play a bit with 20.03 overload.nix by adding `recurseIntoAttrs` to imports, but then it fails because of the missing `lib.kernel` which only appears with 20.09.

**Comment by henrytill** [comment]
> Running same on `nixos-unstable` (20.09) seem to work. Nix builds the rt linux kernel, but it fails on boot with `bug using smp_processor_id() in preemptible code` (probably another issue?)

Neither @magnetophon or I have this problem on unstable, so it might be something specific to your setup.

**Comment by henrytill** [comment]
Okay to close after #124?
